docs/next_steps.md

```diff
@@ -20,6 +20,56 @@ platform from the list of provisioned ones. Otherwise, when one machine is
 decommissioned, all others would have to be provisioned again after clearing
 Fobnail's memory.
 
+## Removing MAC address from platform metadata
+
+Currently, MAC address of the primary NIC is used as part of platform metadata
+described [here](https://fobnail.3mdeb.com/fobnail-api/#adminprovisionidmeta).
+Determining which NIC is the primary one has been problematic due to lack of
+standardized way to do so - procedure depends on how the NIC is connected, this
+should be PCI on virtually any x86 platform, but not on other architectures.
+
+- TODO: we could discuss other changes to platform metadata if so desired.
```

Comment:
I agree that there are problems with obtaining MAC. On the other hand, in some cases this was the most unique part of metadata, especially on boards with coreboot that often [hardcodes serial number](https://review.coreboot.org/plugins/gitiles/coreboot/+/refs/heads/master/src/arch/x86/smbios_defaults.c#40) to [123456789](https://review.coreboot.org/plugins/gitiles/coreboot/+/refs/heads/master/src/Kconfig#897) unless someone bothers to [implement it properly](https://review.coreboot.org/plugins/gitiles/coreboot/+/refs/heads/master/src/mainboard/pcengines/apu2/mainboard.c#264).